lib/flare/flare_flame.dart

```diff
@@ -0,0 +1,172 @@
+import 'dart:math';
+import 'dart:ui';
+
+import 'package:flame/components/component.dart';
+
+import 'package:flutter/rendering.dart';
+import 'package:flutter/services.dart';
+
+import 'package:flare_flutter/asset_provider.dart';
+import 'package:flare_flutter/flare_actor.dart';
+import 'package:flare_flutter/flare_controller.dart';
+import 'package:flare_flutter/provider/asset_flare.dart';
+
+import 'package:flare_dart/math/aabb.dart';
+import 'package:flare_dart/math/mat2d.dart';
+import 'package:flare_dart/math/vec2d.dart';
+
+class FlareActorPipelineOwner extends PipelineOwner {}
+
+class FlareActorComponent extends PositionComponent {
```

Comment:
The file should maybe be named something that reflects this?

Reply:
Wilco